src/mmw/js/src/modeling/models.js

```diff
@@ -108,16 +108,23 @@ var ProjectModel = Backbone.Model.extend({
         this.set('name', newName);
     },
 
+    // Flag to prevent double POSTing of a project.
+    saveCalled: false,
```

Comment:
From the docs
```
isNewmodel.isNew() 
Has this model been saved to the server yet? If the model does not yet have an id, it is considered to be new.
```
In our case we don't have an id but we have a pending request. So if we save again we get the double POST, we really want to create something like a lock until the first save comes back.

To be fair though I did not try it and assumed that checking isNew was about the same as checking if this.get('id')